### PR TITLE
HttpError: export base component

### PR DESCRIPTION
**Pull request by NRodriguezcuellar**
Export the base HttpError component for customizability

### CODE REVIEW

File: src/lib/components/HttpErrors/index.js [hunk 1]
```diff
@@ -5,3 +5,5 @@ export {
   TooManyRequests,
   InternalServerError,
 } from './HttpErrors';
+
```
Comment (kprzerwa) [inline src/lib/components/HttpErrors/index.js:8]: couldn't it be moved in the same export as the ones on top?

File: src/lib/components/HttpErrors/HttpError.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ import { Grid, Button, Icon } from 'semantic-ui-react';
 import PropTypes from 'prop-types';
 import Overridable from 'react-overridable';
 
-class HttpError extends Component {
+export class HttpError extends Component {
```
Comment (kprzerwa) [inline src/lib/components/HttpErrors/HttpError.js:7]: the class is already exported from this file (check the last line of this file)

Comment (NRodriguezcuellar) [reply src/lib/components/HttpErrors/HttpError.js:7]: i know, but it does not work me if i do not export it like this

Comment (kprzerwa) [reply src/lib/components/HttpErrors/HttpError.js:7]: I understand but it is not a correct solution, double export is misleading

File: src/lib/components/HttpErrors/index.js [hunk 3]
```diff
@@ -5,3 +5,5 @@ export {
...
+export { HttpError } from './HttpError';
```
Comment (kprzerwa) [inline src/lib/components/HttpErrors/index.js:9]: did you try 
```suggestion
export { default as HttpError } from './HttpError';
```

Comment (NRodriguezcuellar) [reply src/lib/components/HttpErrors/index.js]: lets try

Comment (NRodriguezcuellar) [reply src/lib/components/HttpErrors/index.js]: looks like that does the trick

**Review by ntarocco — APPROVED** [review]

**Pull request merged**